src/routes/+layout.svelte

```diff
@@ -17,36 +17,42 @@
     import { LightSwitch } from "@skeletonlabs/skeleton";
 </script>
 
-<AppBar
-    gridColumns="grid-cols-3"
-    slotDefault="place-self-center"
-    slotTrail="place-content-end"
->
-    <svelte:fragment slot="lead">
-        <img src="/menuIcon.svg" alt="menu icon" />
-    </svelte:fragment>
-    <svelte:fragment slot="default">
-        <a href="/">
-            <!-- Voycar Logo -->
-            <img src={Logo} alt="logo" class="w-500" style="scale: 0.5;" />
-        </a>
-    </svelte:fragment>
-    <svelte:fragment slot="trail">
-        <a href="/user">
-            <!-- User icon -->
-            <img src="/userIcon.svg" alt="user icon" />
-        </a>
-        <LightSwitch />
-    </svelte:fragment>
-</AppBar>
-<TabGroup>
-    <TabAnchor href="/" selected={$page.url.pathname === "/"}>Home</TabAnchor>
-    <TabAnchor
-        href="/reservations"
-        selected={$page.url.pathname === "/reservations"}
-        >Reservierungen</TabAnchor
+<div>
+    <AppBar
+        gridColumns="grid-cols-3"
+        slotDefault="place-self-center"
+        slotTrail="place-content-end"
     >
-</TabGroup>
-<div class="p-4">
-    <slot />
+        <svelte:fragment slot="lead">
+            <img src="/menuIcon.svg" alt="menu icon" />
+        </svelte:fragment>
+        <svelte:fragment slot="default">
+            <a href="/">
+                <!-- Voycar Logo -->
+                <img src={Logo} alt="logo" class="w-500" style="scale: 0.5;" />
```

Comment:
Hier können wir die entsprechende Tailwind [utility](https://tailwindcss.com/docs/scale#basic-usage) nutzen
```suggestion
                <img src={Logo} alt="logo" class="scale-50 w-500"/>
```